.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx --no grunt configure
+npx --no lint-staged
```

Comment:
I think it's a safe requirement to say if you are commit code you need to have dev dependencies installed. `lint-staged` also runs prettier and eslint, which are required dev dependencies 

```suggestion
npx grunt configure
npx lint-staged
```